examples/uccf12/uccf12.cpp

```diff
@@ -47,7 +48,7 @@ try_main() {
     wick.spinfree(false)
         .full_contractions(false);
     auto result = wick.compute();
-    simplify(result);
+   simplify(result);
```

Comment:
formatting seems off here and elsewhere ... do you not have auto-formatting ON in IDE?